integration/client/container_test.go

```diff
@@ -174,6 +175,146 @@ func TestContainerStart(t *testing.T) {
 	}
 }
 
+func readShimPath(taskID string) (string, error) {
+	runtime := fmt.Sprintf("%s.%s", plugin.RuntimePluginV2, "task")
+	shimBinaryNamePath := filepath.Join(defaultState, runtime, testNamespace, taskID, "shim-binary-path")
+
+	shimPath, err := os.ReadFile(shimBinaryNamePath)
+	if err != nil {
+		return "", err
+	}
+	return string(shimPath), nil
+}
+
+func copyShim(shimPath string) (string, error) {
```

Comment:
Probably you can just use https://pkg.go.dev/github.com/containerd/continuity@v0.3.0/fs#CopyFile

Reply:
Updated, Thanks